dojo/settings/settings.dist.py

```diff
@@ -1123,6 +1123,7 @@ def saml2_attrib_map_format(dict):
     'Hydra Scan': ['title', 'description'],
     'DrHeader JSON Importer': ['title', 'description'],
     'PWN SAST': ['title', 'description'],
+    'Whispers': ['vuln_id_from_tool'],
```

Comment:
```suggestion
    'Whispers': ['vuln_id_from_tool', 'file_path', 'line'],
```
I have the feeling that this version should be better.

Reply:
added